### PR TITLE
galen-bin v1.4.10

### PR DESCRIPTION
**Pull request by alexandregz**
New formula:
Galen Framework (http://galenframework.com/) 

PS: I'm not associate with that, despite the insistence :smiley:

### CODE REVIEW

File: Library/Formula/galen-bin.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    EOS
+  end
+
+  def install
+    libexec.install "galen.jar"
+    (bin/"galen").write(galen_wrapper)
+  end
+
+  test do
+    output = shell_output("#{bin}/galen -v")
```
Comment (mikemcquaid) [inline Library/Formula/galen-bin.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

Comment (alexandregz) [inline Library/Formula/galen-bin.rb]: You're right, I'm going to modify with a real use. Thx for the feedback.

**Comment by alexandregz** [comment]
brew-test-bot failed because test machine haven't firefox installed:

```
Testing galen-bin
==> /usr/local/Cellar/galen-bin/1.4.10/bin/galen test /private/tmp/galen-bin-RxQHHX/my.test
org.openqa.selenium.WebDriverException: Cannot find firefox binary in PATH. Make sure firefox is installed. OS appears to be: MAC
```

Do I change the test to just one simple "-v" like previously or formula needs to control firefox binary?

**Comment by alexandregz** [comment]
Brew-test-bot previously failed because test machine haven't firefox installed: 

```
Testing galen-bin
==> /usr/local/Cellar/galen-bin/1.4.10/bin/galen test /private/tmp/galen-bin-RxQHHX/my.test
org.openqa.selenium.WebDriverException: Cannot find firefox binary in PATH. Make sure firefox is installed. OS appears to be: MAC
Build info: version: 'unknown', revision: 'unknown', time: 'unknown'
```

I've modified test to control that. Without firefox installed, test is just "galen -v" command (it works but isn't useful)

**Comment by mikemcquaid** [comment]
> I've modified test to control that. Without firefox installed, test is just "galen -v" command (it works but isn't useful)

Yeh, let's just go for that then I guess. You can remove the other `if` because the test is mostly just for the bots anyway. Sorry for the goose chase!

**Comment by alexandregz** [comment]
Done (if I have understood correctly :blush:)

File: Library/Formula/galen-bin.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
+class GalenBin < Formula
```
Comment (mikemcquaid) [inline Library/Formula/galen-bin.rb]: Sorry, just noticed this: should this be maybe called just `galen`?

Comment (alexandregz) [inline Library/Formula/galen-bin.rb]: yep!

**Comment by mikemcquaid** [comment]
Thanks for your great work here!

**Comment by alexandregz** [comment]
you're welcome!

thx for you patience!

**Comment by alexandregz** [comment]
Sorry, now Formula uses source to make .jar (and it's a new version)

https://github.com/Homebrew/homebrew/pull/36219

**Pull request closed without merging**